torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -477,6 +477,16 @@ std::shared_ptr<FutureMessage> RequestCallbackImpl::processMessage(
             // processMessage().
             ClearAutogradContextGuard guard;
             processRpc(*rpc, messageType, id, retFuture);
+          } catch (py::error_already_set& e) {
```

Comment:
Is this the only error type we gonna get from Python?

Reply:
Yes, this is pybind implementation. If C++ calls pybind, and a Python exception is thrown and not handled in Python, this is the only exception that will be thrown to C++ land.

Reply:
what is the scope of things that can be called from procesRpc?  We don't always call C++ through pybind in PyTorch.